docs/conf.py

```diff
@@ -115,7 +115,7 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'TungstenCIDashboard', 'Tungsten CI dashboard Documentation',
+    (master_doc, 'TungstenCIDashboard', 'Tungsten CI acid Documentation',
```

Comment:
Rollback this change. It's out of scope and in collision with https://github.com/codilime/acid/pull/1